src/client/polyfills/css-shim/scope.ts

```diff
@@ -17,7 +17,7 @@ export function parseCSS(original: string): CSSScope {
 }
 
 export function addGlobalStyle(globalScopes: CSSScope[], styleEl: HTMLStyleElement) {
-  const css = parseCSS(styleEl.innerHTML);
+  const css = parseCSS(styleEl.textContent || '');
```

Comment:
To ensure that `parseCSS()` always receives String value as we don't have 100% control on `textContent` property.